provider/resources.go

```diff
@@ -214,6 +214,9 @@ func Provider() tfbridge.ProviderInfo {
 					},
 				},
 			},
+			"google_access_context_manager_access_level_condition": {
+				Tok: gcpResource(gcpAccessContextManager, "AccessLevelsCondition"),
```

Comment:
Should be "level"?
```suggestion
				Tok: gcpResource(gcpAccessContextManager, "AccessLevelCondition"),
```

Reply:
Fixed and a rebuild is underway! Great catch